gateway/src/apicast/policy/jwt_claim_check/readme.md

```diff
@@ -0,0 +1,134 @@
+# JWT claim Check Policy
+
```

Comment:
I think we need to add a note somewhere in this document like the one we added in the routing policy when using JWT: `In order to be able to route based on the value of a jwt claim, there needs to be a policy in the chain that validates the jwt and stores it in the context that the policies share.`: https://github.com/3scale/APIcast/tree/master/gateway/src/apicast/policy/routing#rule-that-uses-a-jwt-claim